src/Search/Api/Elasticsearch/Command/resources/_default_.yaml

```diff
@@ -44,3 +44,13 @@ _default_:
       type: date
       format: strict_date_optional_time
       index: not_analyzed
+
+    snippet:
+      type: object
+      properties:
+        format:
+          type: string
+          index: not_analyzed
```

Comment:
seems like the right intent to prevent as much work from Elasticsearch as possible. The index will already be heavy